src/body.rs

```diff
@@ -160,7 +159,7 @@ impl Body {
     /// # Examples
     ///
     /// ```
-    /// # fn main() -> Result<(), http_types::Error> { async_std::task::block_on(async {
+    /// # fn main() -> Result<(), http_types::Error> { futures_lite::future::block_on(async {
```

Comment:
Instead of relying on `block_on` (which `futures_util` does not have), these should move to using `#[async_std::main]`
```suggestion
    /// # #[async_std::main]
    /// # async fn main() -> Result<(), http_types::Error> {
```

Reply:
Replaced with `async_std::task::block_on` since that was already being used.

Reply:
These should still be moved to `#[async_std::main]`, either in this PR or in another.

Reply:
Using `#[async_std::main]` does increase compile times though. It's probably better to make that change in a separate PR.

Reply:
Ah I suppose we may want to leave the hidden ones as is then...